Source/g_game.c

```diff
@@ -2226,6 +2228,32 @@ static int G_GetHelpers(void)
   return j ? j+1 < myargc ? atoi(myargv[j+1]) : 1 : default_dogs;
 }
 
+// [FG] support named complevels on the command line, e.g. "-complevel boom",
+
+static int G_GetNamedComplevel (const char *arg)
+{
+  int i;
+
+  const struct {
+    int level;
+    const char *const name;
+  } named_complevel[] = {
+    {109, "vanilla"},
```

Comment:
We should also add "2", "3", "4", "9" and "11" as aliases as well as the IWAD names. And we also have the concept of `gameversion` which should be taken into consideration in the "vanilla" case. Should choice of `-complevel` override `-gameversion` here if vice versa?

Reply:
> Should choice of `-complevel` override `-gameversion`

I think it should. Did it in https://github.com/fabiangreffrath/woof/pull/157/commits/2ee34ac10ce5f64acbf47cfd56c869cd2ea4c1fc